app/models.py

```diff
@@ -342,6 +342,7 @@ class Service(db.Model, Versioned):
     )
     crown = db.Column(db.Boolean, index=False, nullable=False, default=True)
     rate_limit = db.Column(db.Integer, index=False, nullable=False, default=3000)
+    contact_link = db.Column(db.String(255), nullable=True, unique=False)
```

Comment:
Will it be possible to have more than one contact link? For example is it not possible to have more than one template that includes a document download link. 
Just wondering if this should be in a mapping table. Maybe I'm over thinking it.

Reply:
When I discussed it with Chris H it was a Service contact link i.e. 'Check your state pension' would have a contact page. I think in MVP terms this is right - per template feels like an advanced feature we should user research or test etc.